_plugins/author.rb

```diff
@@ -97,8 +98,9 @@ def lookup(input, args)
     def team_link(input)
       team = Jekyll.sites[0].collections['team'].docs
       index = team.find_index {|x| x.data['name'] == input}
+      baseurl = Jekyll.sites[0].config['baseurl']
```

Comment:
Useless assignment to variable - `baseurl`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.